tswow-install/bin/scripts/data/Settings.d.ts

```diff
@@ -0,0 +1,628 @@
+import { WDirectory } from "../util/FileTree";
```

Comment:
(applies to all tswow-install files) these files should not be source-controlled, looks accidental (usually tswow-install is outside source directory, so i forgot to add it to .gitignore)